src/components/detail/Detail.vue

```diff
@@ -232,38 +112,37 @@ export default {
     },
 
     methods: {
-        filterChartTypes () {
-            const self = this
-
-            self.chartTypes = self.availableChartTypes.filter(function (c) {
-                if (!self.metricData) { return false; }
-                if (self.metricData.type === 'bars') { return c.chart !== 'line' }
-                if (self.metricData.type === 'lines') { return c.chart === 'line' }
-                return c.chart === 'table'
-            })
-
-            self.chartType = self.chartTypes[0].chart
-            self.chartIcon = self.chartTypes[0].icon
-        },
 
         wikiSelected (wiki) {
-            this.$emit('wiki', wiki)
+            this.$emit('wiki', wiki);
+            this.loadData();
         },
         loadData () {
-            const self = this
-
             this.highlightMetric = { name: this.metric, area: this.area }
 
-            config.metricData(this.metric, this.area).then(function (result) {
-                self.metricData = result
-                self.breakdowns = result.breakdowns
-            })
-
-            config.metrics(this.area).then(function (result) {
+            config.metricData(this.metric, this.area).then((result) => {
+                this.metricData = result
+                this.breakdowns = result.breakdowns
+                let aqsApi = new AQS();
```

Comment:
should instantiate this just once in case it has promises and stuff we wanna keep around.  That should be handled by browser cache, but for example the sitematrix does all that fancy computation, so no reason to do that twice.